WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+<br> Clairvoyance
+<br> Counterspell
+<br> Create food and Water
+<br> Dispel Magic
+<br> Feign Death
+<br> Fireball
+<br> Fly
+<br> Galder's Tower ^LLK^
+<br> Glyph of Warding
+
+\columnbreak
+
+<br> Haste
+<br> Hypnotic Pattern
+<br> Intellect Fortress ^TCE^
+<br> Magic Circle
+<br> Major Image
+<br> Nondetection
+<br> Protection from Energy
+<br> Pulse Wave ^EGW^
+<br> Remove Curse
+<br> Sending
+<br> Sleet Storm
+<br> Slow
+<br> ✦ Spellsteal
+<br> Tidal Wave ^XGE^
+<br> Tiny Servant ^XGE^
+<br> Tongues
+
+##### 4th Level
+Arcane Eye
+<br> ✦ Amplify or <br>&nbsp;&nbsp;&nbsp; Dampen Magic
+<br> ✦ Arcane Barrage
+<br> Banishment
+<br> Charm Monster ^XGE^
+<br> Confusion
+<br> Conjure Minor Elementals
+<br> Dimension Door
+<br> Fire Shield
+<br> Galder's <br>&nbsp;&nbsp;&nbsp; Speedy Courier ^LLK^
+<br> Gravity Sinkhole ^EGW^
+<br> Greater Invisibility
+<br> Hallucinatory Terrain
+<br> ✦ Ice Block
+<br> Ice Storm
+<br> Leomund's Secret Chest
+<br> Locate Creature
+<br> Mordenkainen's <br>&nbsp;&nbsp;&nbsp; Private Sanctum
+<br> Otiluke's Resilient Sphere
+<br> Polymorph
+<br> Summon Elemental ^TCE^
+<br> Wall of Fire
+
+##### 5th Level
+Animate Objects
+<br> Bigby's Hand
+<br> Cone of Cold
+<br> Conjure Elemental
+<br> Creation
+<br> Dominate Person
+<br> Far Step  ^XGE^
+<br> Flame Strike
+<br> Hold Monster
+<br> Immolation ^XGE^
+<br> Mislead
+<br> Passwall
+<br> Planar Binding
+<br> Scrying
+<br> Seeming
+<br> Skill Empowerment ^XGE^
+<br> Synaptic Static ^XGE^
+<br> Teleportation Circle
+<br> Wall of Force
+
+\columnbreak
+
+##### 6th Level
+Arcane Gate
+<br> Contingency
+<br> Disintegrate
+<br> Globe of Invulnerability
+<br> Gravity Fissure ^EGW^
+<br> Guards and Wards
+<br> Heroes' Feast
+<br> Investiture of Flame ^XGE^
+<br> Investiture of Ice ^XGE^
+<br> Mass Suggestion
+<br> Mental Prison ^XGE^
+<br> Otiluke's Freezing Sphere
+<br> Programmed Illusion
+<br> Scatter ^XGE^
+<br> True Seeing
+<br> Wall of Ice
+<br> Word of Recall
+
+##### 7th Level
+Delayed Blast Fireball
+<br> Etherealness
+<br> Fire Storm
+<br> Forcecage
+<br> Mirage Arcane
+<br> ✦ Glacial Spike
+<br> Mordenkainen's <br>&nbsp;&nbsp;&nbsp; Magnificent Mansion
+<br> Reverse Gravity
+<br> Project Image
+<br> ✦ Pyroblast
+<br> Sequester
+<br> Teleport
+
+##### 8th Level
+Antimagic Field
+<br> ✦ Blast Wave
+<br> Clone
+<br> Demiplane
+<br> ✦ Ice Nova
+<br> Illusory Dragon ^XGE^
+<br> Incendiary Cloud
+<br> Mind Blank
+<br> Reality Break ^EGW^
+<br> Tsunami
+
+##### 9th Level
+Blade of Disaster ^TCE^
+<br> Foresight
+<br> Gate
+<br> Imprisonment
+<br> Invulnerability ^XGE^
+<br> ✦ Jaina's Flying Ship
+<br> Mass Polymorph ^XGE^
+<br> Prismatic Wall
+<br> Meteor Swarm
+<br> Ravenous Void ^EGW^
+<br> Time Ravage
+<br> Time Stop
+<br> True Polymorph
+
+</div>
+
+<div class='footnote'>PART 2 | SPELLS </div>
+
+\pagebreakNum
+
+> ##### Extended spell list 
+> This list represents the spells a mage would be able to learn but are not in line with the Warcraft fantasy for a mage. <br> 
```

Comment:
I want for this heading to elaborate a bit more on why this is, something that captures the feel that ... while a mage could choose to learn spells dealing with demonic and necrotic powers, to do so is highly frowned upon, even forbidden, and may lead to dire consequences?

Reply:
I get it but these spells aren't all just forbidden knowledge. Some aren't really a thing in Warcraft.

Reply:
Maybe tweak it like this, to bring the Dungeon Master into the context? 🤔 

_This list holds spells that are not in line with the Warcraft fantasy for mages, but that a Dungeon Master could still allow a mage to learn._

Or a bit longer, with some context:

_This list holds spells that are not in line with the Warcraft fantasy for mages, but that a Dungeon Master could still allow a mage to learn. Many of these spells deal with necromancy or demonology, both of which are forbidden by most conclaves of magi. Others are powerful Wizard with no real equivalent in the Warcraft universe._